api/managementpb/backup/common.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package backup.v1beta1;
+
+option go_package = "api/managementpb/backup;backupv1beta1";
+
+// Status shows the current status of execution either backup or restore.
+enum Status {
```

Comment:
Maybe it's better to have seperate status for backup/restore.

Reply:
could be helpful, but then we need to change naming of each status e.g. `RESTORE_STATUS_PENDING`, `BACKUP_STATUS_PENDING` since they are in the same package.

Reply:
@artemgavrilov what do you think?

Reply:
Better to have different status sets for backup and restore. I don't think that we will have pending restores, for example.

Reply:
Separate status added.